style.css

```diff
@@ -1,104 +1,552 @@
+/* General selections */
 * {
-  padding: 0;
   margin: 0;
-  box-sizing: border-box;
+  padding: 0;
+}
+
+:root {
+  --navcolor: #3c3a39;
+  --txtcolor: #fff;
+  --txtcolor2: #172b4d;
+  --maincolor: #ff6b00;
+  --bkcolor: #1c1a19;
+  --line: #212121;
+  --colorb: #ffffff3d;
+  --backc: #fbf8f7;
+  --btncol: #f4f1f0;
+  --btncol2: #fff5e1;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
 }
 
-.container {
+/* The navigation selector */
+nav {
   display: flex;
+  flex-direction: row;
   justify-content: space-between;
-  background-color: #3e3c3c;
-  padding: 0.7rem;
-  width: 100%;
+  align-items: center;
+  width: auto;
+  padding: 1.8rem 2rem 1rem 2rem;
+  background: var(--navcolor);
 }
 
-body {
-  background-color: #1c1a19;
+nav > a {
+  text-decoration: none;
+  color: var(--txtcolor);
+  font-family: 'Roboto', sans-serif;
+  font-size: 1rem;
+  font-weight: 700;
 }
 
-h1 {
-  font-style: normal;
-  font-weight: 600;
-  font-size: 45px;
-  color: #f55800;
-  width: 300px;
-  line-height: 1em;
+/* headline selector */
+#headline {
```

Comment:
Please avoid using `id` selectors to style your page. IDs shouldn't be used in selectors because these rules are too tightly coupled with the HTML and have no possibility of reuse. It's much preferred to use classes in selectors and then apply a class to an element on the page. Additionally, IDs impact your specificity and can lead to specificity wars.

- Kindly update all your CSS selectors using IDs and use classes instead. An easy way you can achieve this is by giving all the selected elements a `class` with the same name as the IDs, then simply replace all the `#` with a dot (`.`).

Reply:
thank you already made the changes